openedx/openedx-platform
# docs: add more details to the survey report README

## Description

This PR only adds extra information to the Survey Report Django App in the README.

## Supporting information

Context of the project [FC-0005 - Openedx Survey Report Djangoapp](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3613294658/FC-0005+-+Openedx+Survey+Report+Djangoapp)

## Testing instructions

You can see the GitHub view here: https://github.com/eduNEXT/edx-platform/tree/mfmz/update-readme-survey-report/openedx/features/survey_report#survey-report
